rstream/_pyamqp/_encode.py

```diff
@@ -791,7 +791,11 @@ def encode_unknown(output: bytearray, value: Optional[object], **kwargs: Any) ->
     elif isinstance(value, float):
         encode_double(output, value, **kwargs)
     elif isinstance(value, int):
-        encode_int(output, value, **kwargs)
+        # if the value fits within AMQP 1.0 32-bit signed integer bounds, encode as an int
+        if -2_147_483_648 <= value <= 2_147_483_647:
```

Comment:
May I ask to create two constansts is https://github.com/rabbitmq-community/rstream/blob/master/rstream/constants.py file? something like MAX_INT32 and MIN_INT32 instead of the numbers? . 
Thank you